Library/Homebrew/dev-cmd/tap-new.rb

```diff
@@ -45,33 +45,31 @@ def tap_new
     MARKDOWN
     write_path(tap, "README.md", readme)
 
-    actions = <<~YAML
+    actions_main = <<~YAML
       name: brew test-bot
       on:
         push:
           branches: master
         pull_request:
       jobs:
         test-bot:
-          runs-on: ${{ matrix.os }}
           strategy:
             matrix:
               os: [ubuntu-latest, macOS-latest]
+          runs-on: ${{ matrix.os }}
           steps:
             - name: Set up Homebrew
               id: set-up-homebrew
               uses: Homebrew/actions/setup-homebrew@master
 
             - name: Cache Homebrew Bundler RubyGems
-              id: cache
               uses: actions/cache@v1
               with:
                 path: ${{ steps.set-up-homebrew.outputs.gems-path }}
                 key: ${{ runner.os }}-rubygems-${{ steps.set-up-homebrew.outputs.gems-hash }}
                 restore-keys: ${{ runner.os }}-rubygems-
 
             - name: Install Homebrew Bundler RubyGems
-              if: steps.cache.outputs.cache-hit != 'true'
```

Comment:
I think it's useful to always run this step, as we can see what gems are updated and what gems are used from the cache.

Reply:
I think in Homebrew repositories perhaps but for third-party repos the speed is preferable of using the cache.